src/taggle.js

```diff
@@ -302,9 +308,11 @@
     Taggle.prototype._attachEvents = function() {
         var self = this;
 
-        _on(this.container, 'click', function() {
-            self.input.focus();
-        });
+        if (this.settings.focusInputOnContainerClick) {
+            _on(this.container, 'click', function() {
```

Comment:
I think this setting assumes that `container` is styled as an input. In my case I'm not styling `container` as the input, but rather the actual input itself which I've chosen to make full-width instead of inset within the pseudo-input.